guacml/enums.py

```diff
@@ -1,6 +0,0 @@
-from enum import Enum
-
-ProblemType = Enum('ProblemType', 'BINARY_CLAS MULTI_CLAS REGRESSION')
```

Comment:
I'm not such a big fan of those enums, we never validated by type anyway, and we already have 90% of other settings as strings, so I'd rather we add validation per setting. I agree we need input validation with clear error messages, but I'm not sure enums are the best way to achieve that, also seems like enums are not really that popular in the python world, most big libraries just use strings.